app/controllers/v0/terms_of_use_agreements_controller.rb

```diff
@@ -112,6 +112,13 @@ def terms_authenticate
       raise Common::Exceptions::Unauthorized unless @user_account
     end
 
+    def create_mhv_account
+      return if terms_code_temporary_auth?
+      return unless current_user.va_patient?
+
```

Comment:
@rileyanderson do we want to check for flipper here?

Reply:
making changes. moving back to draft.